src/main/java/mods/railcraft/common/blocks/tracks/behaivor/CollisionHandler.java

```diff
@@ -48,11 +50,24 @@ public void onEntityCollidedWithBlock(World world, BlockPos pos, IBlockState sta
             if (node.getChargeGraph().getCharge() > 2000) {
                 boolean shock = true;
                 ItemStack overalls = getOveralls(entity);
-                if (overalls != null) {
+                ItemStack boots = getRubberBoots(entity);
+                if ((overalls != null) && (boots != null)) {
```

Comment:
Use InvTools.isEmpty(ItemStack) for all future ItemStack nulls checks please. Needed for 1.11.